build/debs/postinst.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+if [ -f "/run/systemd/resolve/resolv.conf" ]; then
```

Comment:
What if the systemd-resolved service is stopped, this file exists and contains old data? We should probably change this check to:
`if systemctl is-active systemd-resolved; then`

Reply:
Agreed. Will update when I've got a moment

Reply:
Resolved in newer commit. Will squash after approval.